File: app/src/main/java/com/example/android/simpledice/ui/AboutActivity.java
package com.example.android.simpledice.ui;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import com.example.android.simpledice.R;

public class AboutActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_about);

        setTitle(R.string.about);
    }
}

File: app/src/main/java/com/example/android/simpledice/ui/AboutActivity.kt
package com.example.android.simpledice.ui

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.example.android.simpledice.R

class AboutActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_about)

        setTitle(R.string.about)
    }
}
